src/threat_model/qualitative/metadata.md

```diff
@@ -0,0 +1,48 @@
+
+The metadata threat model considers two types of adversaries: snapshot adversaries and active observer adversaries. This section describes the two adversaries and the information they have access to.
+
+## Snapshot adversarires
+
+Snapshot adversaries have access to snapshots of the server's persisted state. They can see the metadata of all messages that have been sent and received up to the point of the snapshot and that haven't been deleted from the long-term storage. An example of a snapshot adversary is a government that has seized the server's hard drives, or a provider that is forced to comply with a government request to hand over data.
+
+The main protection against the snapshot adversary is encryption at rest for most relevant metadata. In this instance, encryption at rest means that clients hold the key material and supply it to the server for individual queries. The server can decrypt the data and process the query before re-encrypting the data and deleting the key from memory.
+
+As a consequence, snapshots become largely useless to the adversary since the senders of messages, the group in which messages were sent, etc. are encrypted. The encryption keys are either only held by clients, or are only held temporarily by the server in volatile memory and are therefore not part of the snapshot.
+
+### Data available to snapshot adversaries
+
+Despite the use of encryption at rest, the snapshot adversary can still see the following data:
+
+- For each group on the DS
+  - The rough size of the group
+  - The timestamp of the last time the group was active (e.g. due to a key update)
+- Reserved group IDs
+- AS key material
+  - Credentials of all clients (includes usernames)
+  - AS credentials and key material
+  - OPAQUE key material
+  - Privacy Pass key material
+- The number of messages in the connection queues of all clients (but not the contents of the messages)
+- All user pseudonyms and their associated client pseudonyms
+- All currently available KeyPackages of all clients
+- The number of messages in all client queues and the (padded) length of each message (but not the contents)
+- QS key material
+  - QS signing keys
+  - QS HPKE encryption keys
+
+## Active observer adversaries
+
+Active observer adversaries have full access to the server including its volatile memory in addition to the snapshots.
+
+The main protection against the active observer is that users interact with the services using per-group pseudonyms instead of their real identities. This means that the adversary cannot link any individual action of a user with the user's identity as seen by other users. The adversary can still see the metadata of messages, but cannot link them to the real identities of the users.
+
+### Data available to active observer adversaries
+
+As per definition, active observer adversaries have access to all data on the server at the time of their activity.
+
```

Comment:
We kind of have the same definition twice. It's also not clear if the active observer can manipulate data. On one hand, this is implied but the word 'active', on the other hand it's excluded by the word 'observer'. We should probably discuss this again.

Reply:
As an inspiration: In other areas, "honest", "curious", and "malicious" are terms to identify fully passive, observant, or actively manipulating threat actors.

Reply:
I guess "curious" would be the one here.

Reply:
I looked around a bit an settled on honest-but-curious, which I have seen used before and which seems to fit quite well for our adversarial model.